<?xml version="1.0"?>
<launch>
    <arg name="cam_name" default="No Camera Name" />
    <param name="cam_name" value="$(arg cam_name)"/>
    <node name="$(arg cam_name)_node" pkg="basler_cam" type="cam_node.py" output="screen" clear_params="true"/>
    <!-- <node ns="camera/$(arg cam_name)" pkg="nodelet" type="nodelet" name="rectify_color" args="standalone image_proc/rectify">
        <remap from="image_mono" to="image_raw" /> 
    </node>  -->
</launch>